.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.vs_code
+__pycache__
+venv
```

Comment:
I think all of these are not particularly useful or contribute to the project. Feel free to reintroduce any of those, should you need them